scc/dockerfile-based-builds.hbs.md

```diff
@@ -127,9 +127,25 @@ use of the SCC, OpenShift accepts the pods created to run Kaniko to build
 the container images.
 
 
+> **Note** Such restrictions are due to well-known limitations in how Kaniko
```

Comment:
Is it correct to have the note repeated here - it's already on line 145?
It seems out of context here as the limitation being described here is because of the default unprivileged
SecurityContextConstraints (SCC)?